CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 
 ## New Functionality
 
+- Support Enum type.
```

Comment:
perhaps add a bit more context that it is about enum types in OData v4 in the generated client. Type safety is there and so on.